core/templates/dev/head/pages/exploration-editor-page/improvements-tab/feedback-improvement-task/feedback-improvement-task.directive.html

```diff
@@ -10,5 +10,10 @@
   <span ng-if="getLatestMessage().author !== null">
     by <strong><[getLatestMessage().author]></strong>
   </span>
-  <span>at <[getLocaleAbbreviatedDatetimeString()]></span>
+  <span ng-if="getLocaleAbbreviatedDatetimeString().getFullYear() == 0">
```

Comment:
The interpretation of the original issue is wrong.

@nithusha21 was just suggesting that, if we are showing a date, we say "on", and if we are showing a time, we say "at".

Reply:
Hey @seanlip, my bad. Sorry for such stupid misinterpretation and more importantly sorry for wasting your time. I have made a new commit, hope its not wrong this time. Please tell me the changes required.